ably/rest_client.go

```diff
@@ -615,19 +597,19 @@ func (c *REST) doWithHandle(ctx context.Context, r *request, handle func(*http.R
 				if c.opts.FallbackHosts != nil {
 					fallback = c.opts.FallbackHosts
 				}
-				log.Info("RestClient: trying to fallback with hosts=%v", fallback)
+				c.log.Infof("RestClient: trying to fallback with hosts=%v", fallback)
```

Comment:
Should be `Info` instead of `Infof`

Reply:
Yes, leaving just the `f` versions is probably a good idea.

Reply:
Just to reiterate, this PR just cleans up what's already there. We can do this separately.